spdx/external_document_ref.py

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
@pombredanne Do you want me to add this in `document.py` itself ?

Reply:
Sure.